app/routes/search/results.ts

```diff
@@ -78,7 +78,7 @@ module.exports = [
       auth: false,
       validate: {
         query: Joi.object({
-          searchString: Joi.string().trim().min(1).required(),
+          searchString: Joi.string().trim().min(3).required(),
```

Comment:
I think, yesterday we discussed in sprint planning not to use this min 3 char validation, can we get this removed?

Reply:
Good spot. I think I was experimenting and left it in there accidentally. I will remove it.